DataFormats/PatCandidates/src/classes_def_objects.xml

```diff
@@ -271,9 +271,10 @@
   <ioread sourceClass = "pat::Jet" version="[1-]" targetClass="pat::Jet" source="" target="daughtersTemp_">
     <![CDATA[daughtersTemp_.reset();]]>
   </ioread>
-  <class name="pat::MET"  ClassVersion="15">
-    <version ClassVersion="15" checksum="428901429"/>
+  <class name="pat::MET"  ClassVersion="16">
+    <version ClassVersion="16" checksum="2969659150"/>
```

Comment:
@Dr15Jones @slava77 (also for my own understanding): is it correct that the same ```ClassVersion``` (here "16") can get two different ```checksum```'s in different releases (here 10_2 vs 11_1)?

Reply:
Once a checksum has been assigned a version number it should not be changed. The only exception is when it is deemed that version was never used for any useful jobs.

Reply:
- changes of existing versions in a release are not allowed
- same versions in different releases with different checksums are not allowed

Reply:
Thank you @Dr15Jones and @slava77 

I assume then that, once ```metSumPtUnclustered_``` is renamed as ```sumPtUnclustered_```. as in the master, the very same structure in terms of class members and methods will be there for this backport and the master. Therefore the same version "16" and checksum as the master can be also backported.

Reply:
if for whatever reason we have to increment a version number here, we should make sure to add it to the master as well

Reply:
> if for whatever reason we have to increment a version number here, we should make sure to add it to the master as well

Once class members and methods are sinchronized as suggested [above](https://github.com/cms-sw/cmssw/pull/29276#discussion_r397214919), the same class version can be considered for both releases.

The only difference in the code itself between the master and the 10_2 backport will come from the application of the clang format updates in the master, and not yet in 10_2.

That is not enough to trigger a version number change. (And even if so, then probably both
```
DataFormats/PatCandidates/interface/MET.h
DataFormats/PatCandidates/src/MET.cc
```
can be directly copied from the version in the master, and at that point they will be absolutely identical in the two CMSSW releases).

Reply:
After fixing the inconsistencies in MET.h/MET.cc the version is now consistent with the master. I'll update the PR soon.